services/QuillLMS/client/app/bundles/Teacher/components/progress_reports/student_overview_table.jsx

```diff
@@ -51,102 +43,86 @@ export default class StudentOveriewTable extends React.Component {
     return 'Not Started'
   }
 
-  greenArrow(row) {
-    const { studentId } = this.props;
-    if (row.completed_at) {
-      return (
-        <a href={`/teachers/progress_reports/report_from_classroom_unit_and_activity_and_user/cu/${row.classroom_unit_id}/user/${studentId}/a/${row.activity_id}`}>
-          <img alt="" src="https://assets.quill.org/images/icons/chevron-dark-green.svg" />
-        </a>
-      )
-    }
-  }
-
   score(row) {
     if (row.completed_at && !notLessonsOrDiagnostic(row.activity_classification_id)) {
-      return {content: 'Completed', color: 'blue'}
+      return 'Completed'
     } else if (row.percentage) {
-      return {
-        content: Math.round(row.percentage * 100) + '%',
-        color: gradeColor(parseFloat(row.percentage)),
-        linkColor: 'standard'
-      }
+      return Math.round(row.percentage * 100) + '%'
     } else {
-      return {content: undefined, color: 'unstarted'}
+      return 'Not Started'
     }
   }
 
-  tableRow(row) {
-    const { studentId } = this.props;
+  columns() {
     const { userIsPremium } = this.state;
-    const scoreInfo = this.score(row);
-    const onClickFunction = row.completed_at ? () => window.location.href = `/teachers/progress_reports/report_from_classroom_unit_and_activity_and_user/cu/${row.classroom_unit_id}/user/${studentId}/a/${row.activity_id}` : () => {}
-    const blurIfNotPremium = userIsPremium ?  '' : 'non-premium-blur'
-    return (
-      <tr className={row.completed_at ? 'clickable' : ''} onClick={onClickFunction}>
-        <td className='activity-image'>{this.activityImage(row.activity_classification_id, scoreInfo.color)}</td>
-        <td className='activity-name'>
-          <a className={scoreInfo.linkColor} href={`/activity_sessions/anonymous?activity_id=${row.activity_id}`}>{row.name}</a>
-        </td>
-        <td>{this.completedStatus(row)}</td>
-        {this.scoreContent(scoreInfo, blurIfNotPremium)}
-        <td className={`activity-timespent ${blurIfNotPremium}`}>{getTimeSpent(row.timespent)}</td>
-        <td className='green-arrow'>{this.greenArrow(row)}</td>
-      </tr>
-    )
-  }
-
-  scoreContent(scoreInfo, blurIfNotPremium) {
-    const activityUngraded = scoreInfo.content === 'Completed'
-    if (activityUngraded) {
-      return (
-        <td className={`score ${blurIfNotPremium}`}>
-          <Tooltip
-            tooltipText="This type of activity is not graded."
-            tooltipTriggerText={scoreInfo.content}
-          />
-        </td>
-      )
-    } else {
-      return (
-        <td className={`score ${blurIfNotPremium}`}>{scoreInfo.content}</td>
-      )
-    }
+    const { studentId } = this.props
+    const blurIfNotPremium = userIsPremium ? null : 'non-premium-blur'
+    return ([
+      {
+        Header: 'Activity',
+        accessor: 'name',
+        resizable: false,
+        maxWidth: 660,
```

Comment:
Are headerWidth and maxWidth 660 coupled? If so, I would extract as a constant to make that dependency explicit.